packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -747,6 +747,83 @@ export class KubernetesClient {
     }
   }
 
+  async readNamespacedDeployment(name: string, namespace: string): Promise<V1Deployment | undefined> {
+    let telemetryOptions = {};
+    const k8sAppsApi = this.kubeConfig.makeApiClient(AppsV1Api);
+    try {
+      const res = await k8sAppsApi.readNamespacedDeployment(name, namespace);
+      if (res?.body) {
+        return res.body;
+      } else {
+        return undefined;
+      }
```

Comment:
```suggestion
return res.body;
```

here and everywhere else

Reply:
Had to rebase, so I fixed this at the same time, along with the place I copy/pasted from.